### PR TITLE
Creation of base project

### PR DESCRIPTION
**Pull request by JoelIvanStella**
## Summary
- Creation of base project
## Trello Card
https://trello.com/c/UF8vAl7l

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/wolox_training_db
+spring.datasource.username=postgres
+spring.datasource.password=lala123
```
Comment (torsello) [inline src/main/resources/application.properties:3]: @JoelIvanStella - I understand that in this instance it is not necessary, but normally these keys are usually set as environment variables so they are not exposed in a repository. Maybe you knew this, but if not, advice never hurts :smile:

**Review by torsello — APPROVED** [review]
Good Job!

File: src/main/resources/application.properties (hunk 1 above)
Comment (JoelIvanStella) [reply src/main/resources/application.properties:3]: Yes, thank Mati!

**Review by jdhiguita — APPROVED** [review]
Great job

**Pull request merged**